### PR TITLE
Update NServiceBus 6.1.0

### PR DESCRIPTION
**Pull request by danielmarbach**
Relates to https://github.com/Particular/NServiceBus/pull/4377

Smoke tests SC plugin against 6.1 (currently contains RC1, we can then update this PR when 6.1 final is released)

### CODE REVIEW

**Comment by timbussmann** [comment]
@Particular/container-maintainers 6.1 is now officially released. Feel free to pick up this PR and update it to the final 6.1 package. It's up to you whether you decide to only update the test projects or if you update the main projects build dependency to 6.1 too. In case of the second option, please be careful to update nuspec dependencies only if you make use of newly available 6.1 APIs.

**Comment by adamralph** [comment]
IMHO we should update the test projects only. When the main projects are updated, the nuspec dependency should be updated too.

**Comment by boblangley** [comment]
@adamralph I reverted the update to only include the tests, rebased, and squashed. Please review and merge.

**Comment by boblangley** [comment]
Build is green now, I fixed the code inspection errors.

File: src/NServiceBus.Autofac.AcceptanceTests/When_declaring_a_public_property.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿namespace NServiceBus.Autofac.AcceptanceTests
 {
     using System.Threading.Tasks;
-    using NServiceBus.AcceptanceTesting;
+    using AcceptanceTesting;
```
Comment (adamralph) [inline src/NServiceBus.Autofac.AcceptanceTests/When_declaring_a_public_property.cs]: Why was this change made? Personally, I prefer the full namespaces to be used, but I'm happy to go with whatever convention has been decided for this repo. However, the declarations should be consistent. Those in the next two lines are still using the full namespaces.

**Comment by boblangley** [comment]
I just did what the configured code inspections told me after the build was
rejected for conventions. I have no strong opinion about the matter.

**Comment by adamralph** [comment]
OK, let's just roll with it for now. We can revisit if/when the inconsistency bothers us.

**Pull request merged**